sdk/samples/iot/templates/iothub/test-resources-post.ps1

```diff
@@ -17,6 +17,29 @@ param (
     $RemainingArguments
 )
 
+function waitForActiveHub
+{
+    $retryCount = 0
+    do
+    {
+      $retryCount++
+      Write-Host "AzIotHub is not yet active so sleeping for $retryCount seconds."
+      Start-Sleep -Seconds $retryCount
+
+      # Get the hub as an object
+      $hub_obj = Get-AzIotHub -ResourceGroupName $ResourceGroupName -Name $iothubName
+    }
+    while ($retryCount -lt 8 -and $hub_obj.Properties.State -ne "Active")
```

Comment:
Any particular reason to pick 8? I guess, it's reasonable, total wait time of ~30 seconds?

Reply:
It was left over from before. I think that was an iterative adjustment where 5 didn't work so we increased it a bit.